scenarios/centos-9/edpm_ceph.yml

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Since `make ceph` deploys toy ceph which does not work with EDPM, So can be rename this job to `cifmw-crc-podified-ceph-deployment`.

We can disable the EDPM deployment by setting `deploy_edpm: false` var.